node/src/Operand.cpp

```diff
@@ -22,10 +22,9 @@ namespace node {
 
     Operand::Operand(const Napi::CallbackInfo& info) : Napi::ObjectWrap<Operand>(info) {
         for (size_t i = 0; i < info.Length(); ++i) {
-            Napi::Object operand = info[i].As<Napi::Object>();
-            WEBNN_NODE_ASSERT_AND_RETURN(operand.InstanceOf(Operand::constructor.Value()),
-                                         "The input must be Operand object.");
-            mInputs.push_back(Napi::Persistent(operand));
+            // It may be an operand or an operator.
+            Napi::Object object = info[i].As<Napi::Object>();
```

Comment:
probably you can still check whether the object is `Operand` or `Operator`, can you?

Reply:
Yes, done,thanks! @huningxin